dashboard-frontend/src/helpers/arrays.ts

```diff
@@ -1 +1,8 @@
 export const lastValue = <T>(values: T[]): T => values[values.length - 1];
+
+export const reducer = (arr: any[]) => {
```

Comment:
this should have a more informative name!

Reply:
agree with @nandanrao here, hard to understand what this function is reducing. Can we make an issue to rename it though ad I dont see it as a blocked to merge

Reply:
Updated to `reduceFieldStateToAnObject` and made the test description more informative too.